src/test/globalSetup.test.ts

```diff
@@ -22,15 +22,15 @@ import { activateExtension } from '../shared/utilities/vsCodeUtils'
 import { FakeExtensionContext, FakeMemento } from './fakeExtensionContext'
 import { TestLogger } from './testLogger'
 import * as testUtil from './testUtil'
-import { createTestWindow } from './shared/vscode/window'
+import { getTestWindow, resetTestWindow } from './shared/vscode/window'
 
 const testReportDir = join(__dirname, '../../../.test-reports')
 const testLogOutput = join(testReportDir, 'testLog.log')
 const globalSandbox = sinon.createSandbox()
+const maxTestDuration = 30_000
```

Comment:
Arbitrary max duration for unit tests. 30 seconds is already very generous but we have some tests that are taking ~7-8 seconds on my machine. Those should probably be moved or optimized.